com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/RemoveNewApplicationsSuffixStep.java

```diff
@@ -50,6 +49,8 @@ private void updateConfigurationSubscribers(List<String> appsToProcess, String m
                                                                            .list();
         for (ConfigurationSubscription subscription : subscriptions) {
             if (appsToProcess.contains(subscription.getAppName())) {
+                getStepLogger().debug(Messages.UPDATING_CONFIGURATION_SUBSCRIPTION_0_WITH_NAME_1, subscription.getAppName(),
+                                      BlueGreenApplicationNameSuffix.removeSuffix(subscription.getAppName()));
```

Comment:
Extract `BlueGreenApplicationNameSuffix.removeSuffix(appName)` in a variable and pass it to `updateConfigurationSubscription`. Otherwise, if you change the code in `createNewSubscription` you might forget to also change this message.